data_table.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import dash
+from dash import dash_table, html
+
+df= pd.read_csv('csv/customers-1000.csv')
+df["Subscription Date"] = pd.to_datetime(df["Subscription Date"]).dt.date
```

Comment:
It normally good to add comment on the intention behind some steps, in this case indicate the what and the why like:

```
Parsing the date fields to date objects to compare to the reference date and identify the old and new customers.  
```